Web/Edubase.Web.UI/Views/Downloads/PreparingFilePleaseWait.cshtml

```diff
@@ -24,7 +24,7 @@
         <div class="govuk-grid-column-full">
             <div class="govuk-breadcrumbs">
                 <ol class="govuk-breadcrumbs__list">
-                    <li class="govuk-breadcrumbs__list-item">@Html.ActionLink("Search", "Index", "Search", new { area = "" }, new { @class = "govuk-breadcrumbs__link"})</li>
+                    <li class="govuk-breadcrumbs__list-item">@Html.ActionLink("Search", "Index", "Search", new { area = "" }, new { @class = "govuk-breadcrumbs__link" })</li>
```

Comment:
looks like this file could be excluded from the commit as it was only to introduce whitespace so cleaner to keep the commits as isolated as possible for files that were impacted